xapian-applications/omega/mimemap.tokens

```diff
@@ -193,6 +193,13 @@ rpm		application/x-redhat-package-manager
 # Atom feeds:
 atom		application/atom+xml
 
+# Ebooks:
+fb2		application/x-fictionbook+xml
+fb2.zip		application/x-zip-compressed-fb2
```

Comment:
I don't think that will work - we take everything after the final `.` as the extension to look up, so `foo.fb2.zip` would be looked up as extension `zip`.

It's probably worth addressing that, but I'd probably make a note and come back to it once the basic e-book support is done.

Reply:
Okay, I am going to remove it from the list and add a note [here](https://trac.xapian.org/wiki/GSoC2019/TextExtraction/Notes)